apps/web/src/pages/block/[id].tsx

```diff
@@ -77,7 +80,24 @@ const Block: NextPage = function () {
     );
 
     detailsFields = [
-      { name: "Block Height", value: blockData.number },
+      {
+        name: "Block Height",
+        value: (
+          <div className="flex items-center justify-start gap-4">
+            {blockData.number}
+            {latestBlock && (
+              <NavArrows
+                prev={buildBlockRoute(blockData.number - 1)}
+                next={
+                  latestBlock.number > blockData.number
+                    ? buildBlockRoute(blockData.number + 1)
+                    : undefined
+                }
+              />
```

Comment:
Blobscan only stores blocks that contain blob transactions, so the next block may not be the immediately preceding or following one

Reply:
Done [83dcc29](https://github.com/Blobscan/blobscan/pull/479/commits/83dcc2943977815edb28f3e55070c47de385decf)